### DISCUSSION TITLE
[FEATURE REQUEST]: add additional headers to engines.http_logstash module

### DISCUSSION THREAD

**Discussion by davama** [saltstack/salt#67495] [Feature Requests]
Hello,

**Is your feature request related to a problem? Please describe.**
There is no problem, just want to add capability to send custom headers on the http_logstash engine module

**Describe the solution you'd like**
Include in the `engines.http_logstash` config a `headers` key where one can add their additional headers to send.

**Describe alternatives you've considered**
This feature does not exist for this module. So no alternatives

I have already tested the code change and it works. No breaking changes to current behavior. (tested on v3004.2)
I have the PR ready to go but looks like I need to create an issue ticket first.

Please let me know if I'm missing something.

Best,
Dave

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by davama** [comment]
My commit is here if anyone interested 

https://github.com/davama/salt/commit/effc12a380bdc5681b8fc92dddc785d473cd92a4
